<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="itemTab">
      <property name="toolTip">
       <string>Summary details for item</string>
      </property>
      <attribute name="title">
       <string>Item</string>
      </attribute>
      <layout class="QVBoxLayout" name="detailsTabLayout"/>
     </widget>
     <widget class="QWidget" name="layoutTab">
      <attribute name="title">
       <string>Layout</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTabWidget" name="layoutTabWidget">
         <property name="toolTip">
          <string>Track layout settings</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tracksLayoutTab">
          <attribute name="title">
           <string>Tracks</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="overviewLayoutTab">
          <attribute name="title">
           <string>Overview</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="styleTab">
      <attribute name="title">
       <string>Style</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QTabWidget" name="styleTabWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="wellPlotStyleTab">
          <attribute name="title">
           <string>Well plot style</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="trackStyleTab">
          <attribute name="title">
           <string>Track style</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="curveStyleTab">
          <attribute name="title">
           <string>Curve style</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="scaleTab">
      <attribute name="title">
       <string>Scale</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QWidget" name="tracksRangePlaceholderWidget" native="true"/>
       </item>
       <item>
        <widget class="QGroupBox" name="overviewGroupBox">
         <property name="title">
          <string>Overview</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tracksGroupBox">
         <property name="title">
          <string>Tracks</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>303</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
